docs/tutorials/03-on-build.md

```diff
@@ -0,0 +1,103 @@
+![buildah logo](../../logos/buildah-logo_large.png)
+
+# Buildah Tutorial 3 
+## Using ONBUILD in Buildah 
+
+The purpose of this tutorial is to demonstrate how Buildah can use a Dockerfile with the OnBuild command within it or how it can be used with the `buildah config` command.  The ONBUILD command stores a command in the meta data of the image which is then invoked whn the container is created.
+
+A note about this container images compliant with the [Open Container Initiative](https://www.opencontainers.org/) (OCI) [image specification](https://github.com/opencontainers/image-spec) do not support the ONBUILD instruction.  Therefore only containers that use the Docker format can use this technique.  Buildah is capable of working seamlessly with either OCI or Docker compliant images and containers. 
+
+First step is to install Buildah and optionally Podman.  In short, `buildah run` emulates the RUN command that is found in a Dockerfile while the `podman run` command emulates the `docker run` command. 
+
+## Setup
+
+The following assumes installation on Fedora.
+ 
+Run as root because you will need to be root for running Buildah commands:
+
+    $ sudo -s
+
+Then install Buildah and optionally podman by running:
+
+    # dnf -y install buildah podman
+
+After installing Buildah we can see there are no images installed. The `buildah images` command will list all the images:
+
+    # buildah images
+
+We can also see that there are also no containers by running:
+
+    # buildah containers
+  
+## ONBUILD in a Dockerfile
+
+The following example is thanks to Chris Collins (GitHub @clcollins), the idea is a directory `/bar` will be created in the secondary container only.
+
+First create two Dockerfiles:
+
+```
+$ cat << EOF > Dockerfile
+FROM fedora:28
```

Comment:
Just use :latest
That way this blog does not get old when we are at fedora:32

Reply:
:+1: